### PR TITLE
Update 130.1-fourier-transform-drawing.md

### PR DESCRIPTION
**Pull request by GypsyDangerous**

### CODE REVIEW

**Comment by gruselhaus** [comment]
@GypsyDangerous Hold on. It's seems to be a bug in the test framework.

**Comment by gruselhaus** [comment]
@ meiamsome Could you take a look at this? The `linkUrl ()` function in `/unit_testing/format.js` should distinguish between a "contribution" `link` and a "normal" `link`. Shouldn't it?

**Comment by blinkityblank** [comment]
@GypsyDangerous I believe that it's because when you link to a youtube video, you have to link to it with the field "video_id"
So instead of `url: "https://www.youtube.com/watch?v=jenANk6QR5c"`
it should be: `video_id: "jenANk6QR5c"`

File: _CodingChallenges/130.1-fourier-transform-drawing.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ contributions:
       name: "Ilay Skutelsky"
     url: "https://editor.p5js.org/ilay.skutelsky/full/2gUUfjjaE"
     source: "https://editor.p5js.org/ilay.skutelsky/sketches/2gUUfjjaE"
+  - title: "Drawing user drawings with fourier transforms in processing"
+    author:
+      name: "David Snyder"
+      url: "https://github.com/GypsyDangerous"
+    url: "https://www.youtube.com/watch?v=jenANk6QR5c"
```
Comment (meiamsome) [inline _CodingChallenges/130.1-fourier-transform-drawing.md]: @blinkityblank is correct, the tests require youtube urls like this:

```suggestion
    video_id: "jenANk6QR5c"
```

**Comment by meiamsome** [comment]
@gruselhaus It's just a simplification to make writing the tests simpler. `video_id` renders correctly out into the website so forcing it everwhere is the simplest way to do it without having to differentiate when links need to have the checks or not.

**Pull request closed without merging**